kapacitor/content.md

```diff
@@ -0,0 +1,133 @@
+# Kapacitor
+
+Kapacitor is an open source data processing engine written in Go. It can process both stream and batch data.
+
+[Kapacitor Official Documentation](https://docs.influxdata.com/kapacitor/latest/introduction/getting_started/)
+
+%%LOGO%%
+
+## Using this image
+
+### Using the default configuration
+
+Start the Kapacitor container with default options:
+
+```console
+$ docker run -p 9092:9092 \
+    -v $PWD:/var/lib/kapacitor \
+    kapacitor
+```
+
+Modify `$PWD` to the directory where you want to store data associated with the Kapacitor container.
+
+You can also have Docker control the volume mountpoint by using a named volume.
+
+```console
+docker run -p 9092:9092 \
+    -v kapacitor:/var/lib/kapacitor \
+    kapacitor
+```
+
+### Configuration
+
+Kapacitor can be either configured from a config file or using environment variables. To mount a configuration file and use it with the server, you can use this command:
+
+Generate the default configuration file:
+
+```console
+$ docker run --rm kapacitor kapacitord config > kapacitor.conf
+```
+
+Modify the default configuration, which will now be available under `$PWD`. Then start the Kapacitor container.
+
+```console
+$ docker run -p 9092:9092 \
+      -v $PWD:/etc/kapacitor:ro \
+      kapacitord -config /etc/kapacitor/kapacitor.conf
+```
+
+Modify `$PWD` to the directory where you want to store the configuration file.
+
+For environment variables, the format is `KAPACITOR_$SECTION_$NAME`. All dashes (`-`) are replaced with underscores (`_`). If the variable isn't in a section, then omit that part.
+
+Examples:
+
+```console
+KAPACITOR_HOSTNAME=kapacitor
+KAPACITOR_LOGGING_LEVEL=INFO
+KAPACITOR_REPORTING_ENABLED=false
+```
+
+Find more about configuring Kapacitor [here](https://docs.influxdata.com/kapacitor/latest/introduction/installation/)
+
+### Exposed Ports
+
+-	9092 TCP -- HTTP API endpoint
+
+#### Subscriptions
```

Comment:
I found Kapacitor to be the more confusing component when initially looking at TICK.  I don't find this updated section to be really helpful.  It suggests starting influxdb after Kapacitor which seems backwards, and it also uses older style linking vs. the built in service discovery which is inconsistent with the Telegraf docs.  The syntax for environment variable substitution when lists are involved could be explained as it isn't intuitive yet the URLS setting seems to be pretty important.